src/containers/users/UserForm.jsx

```diff
@@ -70,7 +70,8 @@ export class UserForm extends Component {
         validationSchema.validateSync(values, { abortEarly: false });
         return {};
       } catch (error) {
-        return this.getErrorsFromValidationError(error);
+        this.test = '';
```

Comment:
Lets find another way to solve this.  Not cool to have statements that don't do anything.